packages/renderer/src/lib/pod/PodmanPodDetailsSummary.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+import { Link } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+
+import DetailsCell from '../details/DetailsCell.svelte';
+import DetailsSubtitle from '../details/DetailsSubtitle.svelte';
+import DetailsTable from '../details/DetailsTable.svelte';
+import DetailsTitle from '../details/DetailsTitle.svelte';
+import type { PodInfoUI } from './PodInfoUI';
+
+export let pod: PodInfoUI;
+let creationTime: Date = new Date(pod.created);
+</script>
+
+<div class="flex px-5 py-4 flex-col items-start h-full overflow-auto">
+  {#if pod}
+    <DetailsTable>
+      <tr>
+        <DetailsTitle>Details</DetailsTitle>
+      </tr>
+      <tr>
+        <DetailsCell>Name</DetailsCell>
+        <DetailsCell>{pod.name}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>ID</DetailsCell>
+        <DetailsCell>{pod.id}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>Created</DetailsCell>
+        <DetailsCell>{creationTime}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>Age</DetailsCell>
+        <DetailsCell>{pod.age}</DetailsCell>
+      </tr>
+      {#if pod}
```

Comment:
You seems to already be in a {#if pod}, might not be necessary to cumulate the checks ?